libs/e2e/src/lib/describer/modelAssessment/modelOverview/ensureAllModelOverviewDatasetCohortsViewBasicElementsArePresent.ts

```diff
@@ -69,7 +91,7 @@ export function ensureAllModelOverviewDatasetCohortsViewBasicElementsArePresent(
     });
   });
 
-  if (isNotebookTest) {
+  if (isNotebookTest && !isVision) {
```

Comment:
Is it possible to add isVision in one place or if condition and do all vision-related tests in one place and non-vision-related checks in another?

Reply:
This will help in the readability of the code.

Reply:
Tried merging 2 of the 3 conditions involving `isVision` to improve readability. Now the 2 conditions left are `!sVision`, and `getNumberOfCohorts(datasetShape, includeNewCohort) <= 1 || isVision` inside `isNotebookTest` condition, which are different enough where keeping it as it is seems more readable.